docs/src/how-to/4-develop.md

```diff
@@ -0,0 +1,145 @@
+
+![](../assets/dev-fork.png)
+![](../assets/dev-fork-create.png)
+
+Now, your fork of KomaMRI should look similar to the image below:
+
+![](../assets/dev-fork-github.png)
+
+## Setup your Local Git Project
+
+Now, you need to clone your fork of the KomaMRI repository. In Julia, you can do this by opening a Julia REPL, open the package manager with the `]` button and typing the following command (make sure to replace `<github-username>` with your GitHub username):"
+```julia-repl
+(@v1.10) pkg> dev https://github.com/<github-username>/KomaMRI.jl
+```
+
+This saves the local Git project at the `~/.julia/dev/KomaMRI/` directory. You can then open VSCode and navigate to this folder; it should look like this:
+
+![](../assets/dev-vscode-git-project.png)
+
+## Setup the Julia Environment
+
+To run KomaMRI locally, you need to set up the Julia environment for this project. The environment's information is ultimately defined by the `Manifest.toml` file. However, since you don't have that file yet, you'll need to create it using the information from the `Project.toml` file.
+
+Let's select the environment for your KomaMRI project:
+
+![](../assets/dev-vscode-select-komamri-environment.png)
+
+Now, open a Julia REPL:
+
+![](../assets/dev-open-julia-repl.png)
+
+In the Julia REPL, you have to create the `Manifest.toml` file with the `develop` command, considering the local folders `KomaMRIBase`, `KomaMRICore`, `KomaMRIFiles`, and `KomaMRIPlots`. Additionally, you can install all the dependencies listed in the `Manifest.toml` with the `instantiate` command:
+```julia-repl
+(KomaMRI) pkg> dev ./KomaMRIBase ./KomaMRICore ./KomaMRIFiles ./KomaMRIPlots
+
+(KomaMRI) pkg> instantiate
+```
+
+With these steps, your environment has been configured.
+
+## Run KomaMRI Locally
+
+You can now run the KomaMRI's user interface:
+```julia-repl
+julia> using KomaMRI
+
+julia> KomaUI()
+```
+![](../assets/dev-launch-ui.png)
+
+Let's see what the default phantom is in the user interface by clicking on the `Phantom dropdown` and then pressing the `View Phantom` button:
+
+![](../assets/dev-default-phantom-brain.png) 
+
+You can close the KomaMRI's user interface.
+
+## Make Changes and Upload Them
+
+So far so good. However, if you want to make some changes to your GitHub Fork or even contribute to the KomaMRI project, you'll need to make some changes to the local Git repository. In the following example, we will change the default phantom displayed in the user interface.
+
+First, create a branch with a descriptive name. For example, you can use `patch-ui-default-phantom` as the branch name:
+
+![](../assets/dev-branch.png)
+![](../assets/dev-branch-name.png)
+
+In this new branch, we are going to edit the `setup_phantom` function located in the file `src/ui/ExportUIFunctions.jl`. Specifically, we will replace the line where a brain phantom is defined with the function `pelvis_brain2D`:
```

Comment:
I believe this should be `pelvis_phantom2D` instead of `pelvis_brain2D`.
```suggestion
In this new branch, we are going to edit the `setup_phantom` function located in the file `src/ui/ExportUIFunctions.jl`. Specifically, we will replace the line where a brain phantom is defined with the function `pelvis_phantom2D`:
```